examples/forms.amp.html

```diff
@@ -72,7 +86,8 @@
 <form method="post"
       action="/form/html/post"
       action-xhr="/form/echo-json/post"
-      target="_blank">
+      target="_blank"
+      on="submit:lightbox1">
```

Comment:
Ok. In that case, we should just add "submit" even right away, right where we add `this.addEvent('tap');`. Any reason not to do it there?